### PR TITLE
Rule update: ARIA attribute is valid (5f99a7)

### PR DESCRIPTION
**Pull request by john-urbilog**
<< Describe the changes >>

Closes issue: 549   (1/5)

## Guidance for the PR (pull request) creator

**When creating PR:**
- [ ] Make sure you requesting to **pull a issue/feature/bugfix branch** (right side) to the master branch (left side).

**After creating PR:**
- [ ] Add yourself (and co-authors) as "Assignees" for PR
- [ ] Add label to indicate if it's a `Rule`, `Definition` or `Chore` (more to the administrative side)
- [ ] Add relevant project (e.g. "Q3 2018 Status") to PR
- [ ] OPTIONAL: If you want anyone in particular to review your pull request, assign them as "Reviewers".
- [ ] Close the issue that the PR resolves (and make sure the issue is referenced in the top of this comment)

# How to Review And Approve
- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines. 
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.

### CODE REVIEW

File: _rules/aria-attr-valid-5f99a7.md [hunk 1]
```diff
@@ -26,7 +26,10 @@ Each target attribute is defined in [WAI ARIA 1.1](https://www.w3.org/TR/wai-ari
 
 ## Assumptions
 
-The ARIA `attribute` is being used to comply to WCAG.
+The rule assumes that an ARIA `attribute`  is necessary to render accessible the element or its content.  This rule checks only for the validity of the attribut in question, but does not check its syntactic validity in the tag where used, nor whether or not it is really necessary.   For example an aria-label in an input which already has a label tag correctly associated is nevertheless valid.   The rule assums that the use of the aria attribute is based on WCAG compliance.
```
Comment (jkodu) [inline _rules/aria-attr-valid-5f99a7.md]: typo: `attribut` > `attribute`

Comment (jkodu) [inline _rules/aria-attr-valid-5f99a7.md]: syntactic validity `in` > `of` the tag

Comment (jkodu) [inline _rules/aria-attr-valid-5f99a7.md]: please surround `aria-label` with back ticks. This helps markdown transformed to html with a `<code>` tag, which helps readability.

File: _rules/aria-attr-valid-5f99a7.md [hunk 2]
```diff
@@ -26,7 +26,10 @@ Each target attribute is defined in [WAI ARIA 1.1](https://www.w3.org/TR/wai-ari
...
+
+[comment : should we not actually be checking this ?]
```
Comment (jkodu) [inline _rules/aria-attr-valid-5f99a7.md]: intentionally left here???

**Comment by jkodu** [comment]
@john-urbilog any updates on this?

**Comment by WilcoFiers** [comment]
I'm going to close this PR. Because the accessibility requirement of this rule changed, this is no longer relevant. I do like this proposal though, and think this is a good example of how to write an assumption.

**Comment by john-urbilog** [comment]
Hi Wilco, & Jey

I hope that Copenhagen is treating you well.  sorry not to be there and
getting into this with you live.

Will keep up on the update (this was but one of 5 mofifications I made, but
most seemed not to register)

John







*John HICKS*

Consultant en Accessibilité Senior

Docteur en Sciences Cognitives

06 28 06 58 60

*Urbilog*

31, rue Papin

59650 VILLENEUVE D'ASCQ

03 28 55 21 30

*Découvrez Urbilog sur notre site Internet **www.urbilog.fr
<http://www.urbilog.fr>*

*Expertise et accompagnement en accessibilité numérique*

*Rendez vos outils digitaux accessibles à tous !*









Le mer. 25 sept. 2019 à 10:30, Wilco Fiers <notifications@github.com> a
écrit :

**Pull request closed without merging**